src/text_manipulation.py

```diff
@@ -0,0 +1,110 @@
+import random
```

Comment:
Should include some comments to give some explanation above each line as to what it's doing, like
`# the line below gets the first thing from a list`
`temp = this_list[0]`

Reply:
included comments